src/include/reconcile.h

```diff
@@ -51,11 +51,17 @@ struct __wt_rec_chunk {
     WT_ITEM key;
     WT_TIME_AGGREGATE ta;
 
-    /* Saved minimum split-size boundary information. */
-    uint32_t min_entries;
-    uint64_t min_recno;
-    WT_ITEM min_key;
-    WT_TIME_AGGREGATE ta_min;
+    /* Saved split-size boundary information. */
```

Comment:
I prefer to reorganise this like, first info about before split (entries, recno, key, ta), ta_after_split, then total chunk info.

Reply:
This was tricky to do as it makes the comments a bit less clear, I had to separate the idea of "split chunk" from "chunk". Anyway I tried to do it in 8c6de08dc0 so please let me know what you think.